### DISCUSSION TITLE
After enabling simple calendar, all attack roll's numbers disappear in the chat

### DISCUSSION THREAD

**Discussion by Rommel-holmes** [vigoren/foundryvtt-simple-calendar#590] [General]
foundry v11 build 315, dnd 5e 4.04

and after i enabled simple calendar, all player's attack roll numbers disappear in the chat, only the attack item shows. The dice still rolls on the screen. but as you can imagine, after the dice disappears, the number is gone, hard to track.

and that is the only module that conflicts with attack roll. I turned on and off all other modules, and found this module is the problem.

**Comment by vigoren** [comment]
This should be fixed in version 2.4.13 of Simple Calendar. If it's but please let me know!
